Os/test/ut/file/FileRules.cpp

```diff
@@ -170,8 +170,8 @@ void Os::Test::File::Tester::assert_file_opened(const std::string &path, Os::Fil
 
     // When the open mode has been specified assert that is in an exact state
     if (not path.empty() && Os::File::Mode::OPEN_NO_MODE != newly_opened_mode) {
-        // Assert file pointer always at beginning when functional
-        if (functional() ) {
+        // Assert file pointer always at beginning when functional and not append
+        if (functional() && Os::File::Mode::OPEN_APPEND != newly_opened_mode) {
```

Comment:
@kevin-f-ortega I don't think we want this APPEND check anymore right?  APPEND updates on write and so asserting a 0 file pointer is valid here, no?

Reply:
Good point, updated